src/scenes/home/header/burger/burger.css

```diff
@@ -0,0 +1,21 @@
+.burger {
+  display: none;
+}
+
+@media screen and (max-width: 950px) {
+.burger {
+  display: block;
+  position: absolute;
+  right: 0;
```

Comment:
Suggestion - we can remove the `position: absolute` and `right: 0`on this element, and just use `justify-content: space-between` on the flex container (the parent element `<div class="header">`). That would keep the logo on the left and the burger on the right, using only Flexbox.